modules/persistent-storage-csi-external-controllers.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-csi.adoc
+
+[id="external-csi-contollers-{context}"]
+= External CSI controllers
+
+External CSI Controllers is a deployment that deploys one or more pods 
+with three containers:
+
+* An external CSI attacher container translates `attach` and `detach` 
+calls from {product-title} to respective `ControllerPublish` and 
+`ControllerUnpublish` calls to the CSI driver.
+* An external CSI provisioner container that translates `provision` and 
+`delete` calls from {product-title} to respective `CreateVolume` and 
+`DeleteVolume` calls to the CSI driver.
+* A CSI driver container
+
+The CSI attacher and CSI provisioner containers talk to the CSI driver 
```

Comment:
s/talk to/communicate with

Reply:
Adjusted this sentence to use the recommendation.